react/tic-tac-toe/src/app/screens/Game/components/Square/styles.module.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../../../../scss/_colors.scss';
```

Comment:
Does `@import 'src/scss/colors.scss';` work ? I think that CRA 2 comes with `algo magico` that recognizes `src` within the root folder, like an absolute path 🤔